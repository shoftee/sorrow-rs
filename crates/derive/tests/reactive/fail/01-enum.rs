use sorrow_derive::Reactive;

#[derive(Reactive)]
pub enum TestEnum {
    One,
    Two,
}

fn main() {}
