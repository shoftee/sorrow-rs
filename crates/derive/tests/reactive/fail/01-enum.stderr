error: derive(Reactive) is only supported on struct types.
 --> tests/reactive/fail/01-enum.rs:4:5
  |
4 | pub enum TestEnum {
  |     ^^^^
